packages/database/src/migrations/20200828013343-AddMoreLaosSchoolsSchoolIndicatorsSubNationalLevelsMapOverlays-modifies-data.js

```diff
@@ -0,0 +1,224 @@
+'use strict';
```

Comment:
I just added some more overlays that I missed from the task description (which caused this ticket testing failed). They are similar to the other overlays in `School Indicators in District/Province`